tests/helpers/markdown_hotkey_helpers.js

```diff
@@ -63,6 +64,10 @@ export function makeItalicHotkeyEvent(input, start, end) {
     return makeMarkdownHotkeyEvent(input, start, end, Constants.KeyCodes.I);
 }
 
+export function makeLinkHotKeyeyEvent(input, start, end) {
```

Comment:
Can we move the exported functions to a separate utils file? Unless this is just a test helper, but since they are exported I guess they are used in the codebase? I know this was the case before this change but it will really help with the overall structure :)

Reply:
Also, there's a typo in the function name. Could you fix that at the same time?

Reply:
Done.